src/astro/utils/database.py

```diff
@@ -0,0 +1,74 @@
+from airflow.hooks.base import BaseHook
+from airflow.providers.sqlite.hooks.sqlite import SqliteHook
+from sqlalchemy import create_engine, text
+
+from astro.constants import Database
+from astro.utils.dependencies import BigQueryHook, PostgresHook, SnowflakeHook
+
+
+def get_database_name(interface):
```

Comment:
Is there a type hint you can put here?

Reply:
@dimberman sure, I can add - the code has been inconsistent in this way and it was not in our definition of done, but I'll adopt them from now onwards.